client/src/components/StyledCard/StyledCard.js

```diff
@@ -9,36 +9,49 @@ import { INDICATOR_TYPES } from 'utils/constants'
 
 const styles = StyleSheet.create({
   root: {
+    display: 'flex',
     flexDirection: 'row',
     justifyContent: 'space-between',
     alignItems: 'center',
+    maxWidth: '100%',
   },
   left: {
+    display: 'flex',
     flexDirection: 'row',
-    justifyContent: 'space-between',
+    flexShrink: 2,
+  },
+  middle: {
+    paddingLeft: 15,
+    flexShrink: 2,
```

Comment:
What is the flexShrink used for?

Reply:
to make sure that the text component shrinks dynamically based on the size of the button box and image box, lmk if it doesn't work properly